### PR TITLE
Add linux foundation trademark

### LINKED ISSUE #5829: Website does not have the correct trademark disclaimer
As part of our ongoing effort to https://github.com/cncf/techdocs/issues/198, we noticed that the website does not pass the trademark criteria on [CLOMonitor](https://clomonitor.io/search?foundation=cncf&not_passing_check=trademark_disclaimer&page=1).

To fix this:
Head to the source code of the website. In the `<footer>` section, add a **disclaimer** or **link** to the Linux foundation trademark disclaimer page:

**Disclaimer**
```
<footer>
   <p>The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, 
         please see our <a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademark Usage page</a>.
   </p>
</footer>
```

**Link**
```
 <footer>
      <ul>
          <li><a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademarks</a></li>
      </ul>
 </footer>
```

### PR DESCRIPTION
**Pull request by ReToCode**
## Proposed Changes
- Add Linux Foundation trademark link
- Fix cookie-banner link (no need to route it to pre-release any more)

Fixes #5829

/assign @knative/steering-committee 
/assign @knative/technical-oversight-committee

### CODE REVIEW

File: overrides/partials/footer.html [hunk 1]
```diff
@@ -73,8 +73,9 @@
       </div>
 
       <div class="footer-links md-footer-copyright">
+        <a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademarks</a> |
         <a href="https://policies.google.com/privacy">Privacy Policy</a> |
-        <a href="/development/about-analytics-cookies/">Cookie Usage</a> |
+        <a href="/docs/about-analytics-cookies/">Cookie Usage</a> |
```
Comment (psschwei) [inline overrides/partials/footer.html:78]: we should also update this link on L113 below (in the cooke banner)

Comment (ReToCode) [reply overrides/partials/footer.html:78]: Nice catch 👍

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve

/hold
in case this needs more approvals (steering?), feel free to remove as you see fit

**Comment by ReToCode** [comment]
I'm not sure, but as this is related to trademarks, better to have them.

PTAL @salaboy @aliok @puerco @evankanderson @nainaz

**Comment by ReToCode** [comment]
Gentle ping @salaboy @aliok @puerco @evankanderson @nainaz

**Comment by aliok** [comment]
/lgtm
/approve
/unhold

Knative is already under CNCF (and Linux Foundation), so I will simply unhold this PR as we're only adding a link here.

Thanks Reto!

**Pull request merged**